pkg/trace/sampler/probabilistic.go

```diff
@@ -58,6 +62,7 @@ func NewProbabilisticSampler(conf *config.AgentConfig, statsd statsd.ClientInter
 		enabled:                  conf.ProbabilisticSamplerEnabled,
 		hashSeed:                 hashSeedBytes,
 		scaledSamplingPercentage: uint32(conf.ProbabilisticSamplerSamplingPercentage * percentageScaleFactor),
+		samplingPercentage:       float64(conf.ProbabilisticSamplerSamplingPercentage) / 100.,
```

Comment:
I know we keep other percentages as decimal proportions like this (although we erroneously call them "rates" https://github.com/DataDog/datadog-agent/blob/7c349bdc078f032bfe331e201be2ea772e251399/pkg/trace/sampler/scoresampler.go#L94 ).

Is this correct to do here, and if so, is `percentage` the best name we can give it?